### ISSUE TITLE
Khanhas fork issue - Windows 11 blur border persists after blur is disabled

### ISSUE THREAD

**Issue by Cariboudjan** [TheAzack9/FrostedGlass#4]
Below is an example skin

Disabling blur and refreshing the skin will disable the blur but the borders persist unless Rainmeter is restarted. Windows 11 only, Khanhas fork.

```
[Variables]
BlurON=0

[BlurBehind]
Measure=Plugin
Plugin=FrostedGlass
BlurEnabled=#BlurON#
Type=Acrylic
Corner=Round

[Meter]
Meter=String
W=100
H=100
SolidColor=255,55,55,55
LeftMouseUpAction=[!WriteKeyValue Variables BlurON (#BlurON#=1?0:1)][!Refresh]
```

**Comment by Cariboudjan** [comment]
Also to add to this, can we have "Border=None" affect the borders in Windows 11?

**Comment by hikariatama** [comment]
> Also to add to this, can we have "Border=None" affect the borders in Windows 11?

No, we can't. It uses internal blur which already comes with borders - check Explorer for example. Its window also has borders:

![image](https://github.com/TheAzack9/FrostedGlass/assets/36935426/5bfe5566-1351-41e5-a1f1-135b3ed411bf)
